src/test/streams_tests.cpp

```diff
@@ -335,6 +339,57 @@ BOOST_AUTO_TEST_CASE(streams_buffered_file)
     fs::remove("streams_test_tmp");
 }
 
+BOOST_AUTO_TEST_CASE(streams_buffered_file_skip)
+{
+    FILE* file = fsbridge::fopen("streams_test_tmp", "w+b");
+    // The value at each offset is the offset.
```

Comment:
I only understood this comment after I got to L#359.
```suggestion
    // The value at each offset is the byte offset (e.g. byte 1 in the file has the value 0x01, etc).
```